02-turtle.md

```diff
@@ -53,6 +55,7 @@ You can also move the turtle to any position without drawing anything.
 ```
 t.teleport(35, 75) # Move turtle from where it is to the coordinate (35, 75) without drawing anything
```

Comment:
The version of Debian installed by the [autoinstall](https://github.com/livewires/autoinstall) repo comes with Python 3.9 and so the `teleport` function won't be available at LiveWires.